src/gui/qgsmaplayeractionregistry.h

```diff
@@ -77,6 +77,9 @@ class GUI_EXPORT QgsMapLayerAction : public QAction
     //! Return availibity of action
     const Targets &targets() const {return mTargets;}
 
+    //! Return whether only enabled in editable mode
+    bool isEnabledOnlyWhenEditable() const { return mEnabledOnlyWhenEditable; }
```

Comment:
Same here - use an  flags() enum and return that here instead of the bool getter